cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -34,93 +34,114 @@ bundle agent cfe_internal_setup_knowledge
       "$(cfe_internal_hub_vars.docroot)"
       comment => "All files in there should be at least 0644",
       handle => "cfe_internal_setup_knowledge_files_doc_root_2",
-      perms => m("0644"),
+      file_select => cfe_internal_docroot_perms,
+      perms => mog("0440", "root", $(def.cf_apache_group) ),
       depth_search => recurse_exclude("inf");   # see exclude dirs in recurse_exclude() body
 
       "$(cfe_internal_hub_vars.docroot)/.htaccess"
       comment => "Correct up htaccess file in doc root",
       handle => "cfe_internal_setup_knowledge_files_doc_root_htaccess",
-      perms => mog("0644","root","root"),
       copy_from => no_backup_cp("$(sys.workdir)/share/GUI/Apache-htaccess");
 
       "$(cfe_internal_hub_vars.docroot)/hub/."
```

Comment:
I don't think this one is used anymore, I think it was there from Nova 1.0 before it had a database.